### PR TITLE
[DOCS] Add documentation for Calendar Bundle

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12           |
| Related tickets | Loosely connected with https://github.com/Sylius/Sylius/pull/14422                   |
| License         | MIT                                                          |

### CODE REVIEW

File: docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Summary
+=======
+
+Date handling for PHP applications.
```
Comment (arti0090) [inline docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst:4]: Anything else for summary here?

File: docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Bug tracking
+------------
+
+This bundle uses `GitHub issues <https://github.com/Sylius/Calendar/issues>`_.
```
Comment (arti0090) [inline docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst:9]: Should we track here or in Sylius/Sylius ??

Comment (jakubtobiasz) [reply docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst:9]: The code isn't a part of Sylius/Sylius repository, so issues related to this bundle should be tracked on `Sylius/Calendar` repository (IMO).

File: docs/components_and_bundles/bundles/SyliusCalendarBundle/installation.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+.. code-block:: bash
+
+    curl -sS https://getcomposer.org/installer | php
+    php composer.phar require sylius/calendar
+
+Adding required bundles to the kernel
+-------------------------------------
+
+You need to enable the bundle inside the kernel.
```
Comment (jakubtobiasz) [inline docs/components_and_bundles/bundles/SyliusCalendarBundle/installation.rst:23]: We can add a note, when you use `symfony/flex` this step can be skipped.

**Review by jakubtobiasz — APPROVED** [review]

File: docs/components_and_bundles/bundles/SyliusCalendarBundle/index.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
+SyliusCalendarBundle
+====================
+
+Sylius Calendar simple component for PHP applications that abstracts date object creation,
+which the main purpose is to ease testing of date depended features.
```
Comment (lchrusciel) [inline docs/components_and_bundles/bundles/SyliusCalendarBundle/index.rst:5]:
```suggestion
Its primary purpose is to ease testing of date and time dependent features.
```

File: docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst (hunk 1 above)
Comment (lchrusciel) [inline docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst:4]:
```suggestion
Date and Time handling for PHP applications. The primary behaviour of this component is to abstract the creation of new DateTime objects. Therefore, you can intercept it and adjust object configuration. Moreover, such a configuration eases the replacement of the default factory in a testing environment. This bundle comes with out-of-the-box integration with Behat, which allows you to set up specific dates as “now.”
```

File: docs/components_and_bundles/bundles/index.rst [hunk 5]
```diff
@@ -6,6 +6,7 @@ Sylius Bundles Documentation
 
     SyliusAddressingBundle/index
     SyliusAttributeBundle/index
+    SyliusCalendarBundle/index
```
Comment (GSadee) [inline docs/components_and_bundles/bundles/index.rst:9]:
```suggestion
    Calendar/index
```

File: docs/components_and_bundles/bundles/map.rst.inc [hunk 6]
```diff
@@ -1,5 +1,6 @@
 * :doc:`/components_and_bundles/bundles/SyliusAddressingBundle/index`
 * :doc:`/components_and_bundles/bundles/SyliusAttributeBundle/index`
+* :doc:`/components_and_bundles/bundles/SyliusCalendarBundle/index`
```
Comment (GSadee) [inline docs/components_and_bundles/bundles/map.rst.inc:3]:
```suggestion
* :doc:`/components_and_bundles/bundles/Calendar/index`
```

**Comment by jakubtobiasz** [comment]
We're going to kill this bundle, so we can skip this PR.

**Pull request closed without merging**